frontend/src/components/warningToast.tsx

```diff
@@ -0,0 +1,29 @@
+import { Toast, ToastContainer } from "react-bootstrap";
+import timers from "../properties/timers";
+import { capitalize } from "../utility/stringUtil";
+import useTranslation from "next-translate/useTranslation";
+import { useState } from "react";
+
+function WarningToast(props: any) {
```

Comment:
any, Juck! I don't think this function takes any props?
```suggestion
function WarningToast() {
```